Source/SwiftLintFramework/Extensions/Configuration+FileGraphSubtypes.swift

```diff
@@ -95,15 +101,6 @@ internal extension Configuration.FileGraph {
     struct Edge: Hashable {
         var parent: Vertix!
         var child: Vertix!
-
-        internal static func == (lhs: Edge, rhs: Edge) -> Bool {
```

Comment:
We can use the compiler-synthesized implementations for `==` and `hash(into:)` since they're identical to these hand-rolled versions.